src/springfield/entity.py

```diff
@@ -267,10 +273,10 @@ def __len__(self):
         return len(self.__values__)
 
     def iteritems(self):
-        return self.__values__.iteritems()
+        return iter(self.__values__.items())
```

Comment:
Both this and the `list()` on `def items()` was automatically done by futurize, and since I left in the list, I decided to leave this in too. I'm unclear why it was automatically added, for the reason you mentioned.